### DISCUSSION TITLE
how to show all hub functionality

### DISCUSSION THREAD

**Discussion by fahedaljghine** [getcandy/hub#55] [Help]
I am wondering how to show or implement all the functionality such as (languages , currencies , payment , shipment , etc..) ?

**Comment by glennjacobs** [comment]
The hub currently doesn't support everything the core API does. We're looking to expand it to do more, but currently the ones you mention it doesn't cover. Those specific things would be configured by the developer in using Laravel and the database. 

We hope to update the docs soon to cover these different topics.
